book/getting-started/run-a-wallaroo-application-docker.md

```diff
@@ -26,15 +26,15 @@ docker run --rm -it --privileged -p 4000:4000 \
 --name wally \
 wallaroo-labs-docker-wallaroolabs.bintray.io/{{ docker_version_url }}
 {%- language name="Windows Powershell", type="bash" -%}
-docker run --rm -it --privileged -p 4000:4000 |
--v /tmp/wallaroo-docker/wallaroo-src:/src/wallaroo |
--v /tmp/wallaroo-docker/python-virtualenv:/src/python-virtualenv |
---name wally |
+docker run --rm -it --privileged -p 4000:4000 `
+-v c:/wallaroo-docker/wallaroo-src:/src/wallaroo `
+-v c:/wallaroo-docker/python-virtualenv:/src/python-virtualenv `
```

Comment:
We still can't map virtualenv as it will fail to run the setup-env entrypoint (python won't have an execute flag on it). The wallaroo source should be fine since our machida and utlitiy executables come form outside that directory.

Reply:
When you map the volume to C:\... the pip installation will be using that directory on NTFS, which won't preserve execute permission bits. So that python stub script will fail to run.

Reply:
It seems like the issue I see is related to newer versions of docker vs the current stable release. I'm worried that we might have issues in upcoming releases. We'll need to monitor this over time but keeping the flags that work with the stable release is probably best.